src/main/scala/org/broadinstitute/dsde/workbench/sam/api/AdminRoutes.scala

```diff
@@ -187,36 +192,72 @@ trait AdminRoutes extends SecurityDirectives with SamRequestContextDirectives wi
       }
     }
 
-  def adminResourceTypesRoutes(user: SamUser, samRequestContext: SamRequestContext): server.Route =
-    pathPrefix("resourceTypes" / Segment / "policies") { resourceTypeNameToAdminister =>
-      asSamSuperAdmin(user) {
-        withNonAdminResourceType(ResourceTypeName(resourceTypeNameToAdminister)) { resourceTypeToAdminister =>
-          val resource = FullyQualifiedResourceId(resourceTypeAdminName, ResourceId(resourceTypeToAdminister.name.value))
-          pathEndOrSingleSlash {
-            getWithTelemetry(samRequestContext, resourceTypeParam(resourceTypeToAdminister)) {
+  def adminGroupsRoutes(user: SamUser, samRequestContext: SamRequestContext): server.Route =
+    path("groups" / Segment / "supportSummary") { groupName =>
```

Comment:
I think this all needs to be wrapped in `asWorkbenchAdmin` like in `adminUserRoutes`? We really should move up `asWorkbenchAdmin` a level into `adminRoutes` to make it more obvious what is covered by it.

Reply:
This is not intended to be `asWorkbenchAdmin`. It checks the required action internally.

Reply:
Note that `asWorkbenchAdmin` is an older way of checking for admin access. It requires that the user be in the fc-admins google group. The intent for this api is not to use that google group and instead use a sam action granted on `resource_type_admin`. This is similar to `/api/admin/v1/resources/*` end points.